src/compile/mark/encode/position-rect.ts

```diff
@@ -110,6 +111,9 @@ function defaultSizeRef(
       return {value: scaleRange.step - 2};
     }
   }
+  if (!hasFieldDef) {
+    return {signal: `0.8 * ${sizeChannel}`};
```

Comment:
Wonder if we extract `{signal: `0.8 * ${sizeChannel}`}` as the constant. But I could not find a good place to place if we extract...

Reply:
Very good question. For consistency with existing similar style, we can respect `config.scale.bandPaddingInner/ barBandPaddingInner/rectBandPaddingInner`.

Basically, we can think of axis without a field as categorical axis with one unique value. 

So we can do: 

```
const padding = getFirstDefined(bandPaddingInner, mark === 'bar' ? barBandPaddingInner : rectBandPaddingInner);
return {signal: `${1 - padding} * ${sizeChannel}`}; 
```